extension/src/cli/dvc/constants.ts

```diff
@@ -10,11 +10,15 @@ export const TEMP_PLOTS_DIR = join(DOT_DVC, 'tmp', 'plots')
 export const FIELD_SEPARATOR = '::'
 
 export const TEMP_EXP_DIR = join(DOT_DVC, 'tmp', 'exps')
+const TEMP_EXP_RUN_DIR = join(TEMP_EXP_DIR, 'run')
 export const DVCLIVE_ONLY_RUNNING_SIGNAL_FILE = join(
-  TEMP_EXP_DIR,
-  'run',
+  TEMP_EXP_RUN_DIR,
   'DVCLIVE_ONLY'
 )
+export const DVCLIVE_STEP_COMPLETED_SIGNAL_FILE = join(
+  TEMP_EXP_RUN_DIR,
+  'DVCLIVE_STEP_COMPLETED'
+)
```

Comment:
[F] This is the new file that gets written to every time a DVCLive step if completed. It will be a file in the main dvc root for experiments running in the queue as well.